src/Payments/Integrations/CashOnDelivery.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * Cash on Delivery (COD) (core) gateway implementation.
+ *
+ * @package WooCommerce/Blocks
+ * @since 2.10.0
```

Comment:
You can use `@since $VID:$` (here and everywhere you've hardcoded the version in this pull) and it will automatically get replaced with the correct version at the time of release by our version bump script. That allows us to not worry about what the version may or may not be when this is actually released.

Reply:
Cool! What does the substitution - is this something new? Added these in 44f5eb9b4d591cecf3f41b1648a9123f16511f89

Reply:
It was added in #948. :slightly_smiling_face: